components/org.wso2.carbon.business.rules.web/README.md

```diff
@@ -0,0 +1,2154 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+
```

Comment:
Why we have such as big readme file ?

Reply:
Updated and added only necessary content to this file.